Layer/Pdo/Pdo.php

```diff
@@ -194,7 +194,12 @@ public function lastInsertId($name = null)
     public function prepare($statement, array $options = [])
     {
         if (!isset($options[\PDO::ATTR_CURSOR])) {
-            $options[\PDO::ATTR_CURSOR] = \PDO::CURSOR_SCROLL;
+            try {
+                $this->getConnection()->getAttribute(\PDO::ATTR_CURSOR);
+                $options[\PDO::ATTR_CURSOR] = \PDO::CURSOR_SCROLL;
```

Comment:
Hmm, I will try. However, iterator support will no be longer be available if scroll cursor is not enabled. Should we throw an error then?